### PR TITLE
Transformations: Move moving average, regression analysis and format string transformations to public preview

### PR DESCRIPTION
**Pull request by oscarkilhed**
**What is this feature?**

Moves transformation features to public preview.

### CODE REVIEW

File: public/app/features/transformers/docs/content.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const transformationDocsContent: TransformationDocsContentType = {
   - **Alias** - (Optional) Enter the name of your new field. If you leave this blank, then the field will be named to match the calculation.
   - **Replace all fields** - (Optional) Select this option if you want to hide all other fields and display only your calculated field in the visualization.
   
-  > **Note:** **Cumulative functions** and **Window functions** modes are currently in private preview. Grafana Labs offers support on a best-effort basis, and breaking changes might occur prior to the feature being made generally available.
+  > **Note:** **Cumulative functions** and **Window functions** modes are currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the addFieldFromCalculationStatFunctions feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```
Comment (mdvictor) [inline public/app/features/transformers/docs/content.ts:71]: Nit: Should we put the name of the feature flags between quotes?

**Review by mdvictor — APPROVED** [review]

File: public/app/features/transformers/docs/content.ts (hunk 1 above)
Comment (baldm0mma) [reply public/app/features/transformers/docs/content.ts:71]: Or back ticks?

**Review by baldm0mma — APPROVED** [review]
Once everything in drone is successfully building, lgtm.

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ Use this transformation to add a new field calculated from two other fields. Eac
 - **Alias** - (Optional) Enter the name of your new field. If you leave this blank, then the field will be named to match the calculation.
 - **Replace all fields** - (Optional) Select this option if you want to hide all other fields and display only your calculated field in the visualization.
 
-> **Note:** **Cumulative functions** and **Window functions** modes are currently in private preview. Grafana Labs offers support on a best-effort basis, and breaking changes might occur prior to the feature being made generally available.
+> **Note:** **Cumulative functions** and **Window functions** modes are currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the addFieldFromCalculationStatFunctions feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:157]:
```suggestion
> **Note:** **Cumulative functions** and **Window functions** modes are currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the `addFieldFromCalculationStatFunctions` feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 3]
```diff
@@ -519,7 +519,7 @@ Use this transformation to customize the output of a string field. This transfor
 
 This transformation provides a convenient way to standardize and tailor the presentation of string data for better visualization and analysis.
 
-> **Note:** This transformation is currently in private preview. Grafana Labs offers support on a best-effort basis, and breaking changes might occur prior to the feature being made generally available.
+> **Note:** This transformation is currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the formatString feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:522]:
```suggestion
> **Note:** This transformation is currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the `formatString` feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 4]
```diff
@@ -1278,7 +1278,7 @@ There are two different models:
 - **Polynomial regression** - Fits a polynomial function to the data.
   {{< figure src="/static/img/docs/transformations/polynomial-regression.png" class="docs-image--no-shadow" max-width= "1100px" alt="A time series visualization with a curved line representing the polynomial function" >}}
 
-> **Note:** This transformation is currently in private preview. Grafana Labs offers support on a best-effort basis, and breaking changes might occur prior to the feature being made generally available.
+> **Note:** This transformation is currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the regressionTransformation feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:1281]:
```suggestion
> **Note:** This transformation is currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the `regressionTransformation` feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```

**Review by imatwawana — APPROVED** [review]
Just formatted the feature toggle names

**Pull request merged**